### PR TITLE
fix(v2): Add description to run Dependency Analytics report in diagnostic message

### PR DESCRIPTION
**Pull request by dvandra**
**Add description to run Dependency Analytics report in diagnostic message**

E.g.: 
```
moment - 2.15.1 has 1 known security vulnerability and 1 security advisory with 1 having medium severity. Recommendation: use version 2.27.0.

View Dependency Analytics report [ctrl/cmd + shift + D]

Dependency Analytics plugin [Powered by Snyk]
```

### CODE REVIEW

**Comment by arajkumar** [comment]
@dvandra Can you please upload the screenshots with these changes?

**Comment by dvandra** [comment]
@arajkumar here it is:

Screenshot from 2020-06-24 13-55-56

File: src/consumers.ts [hunk 1]
```diff
@@ -134,8 +134,8 @@ class EmptyResultEngine extends AnalysisConsumer implements DiagnosticProducer
             return [{
                 severity: DiagnosticSeverity.Information,
                 range: get_range(this.context.version),
-                message: `Application dependency ${this.context.name.value}-${this.context.version.value} - analysis is pending`,
-                source: 'Dependency Analytics Plugin [Powered by Snyk]'
+                message: `Application dependency ${this.context.name.value}-${this.context.version.value} - analysis is pending \n\nView Dependency Analytics report [ctrl/cmd + shift + D]`,
+                source: '\n\n Dependency Analytics plugin [Powered by Snyk]'
```
Comment (arajkumar) [inline src/consumers.ts:138]: Why `\n\n` is added in the front? Can't be added at the end of `message` property?

File: src/consumers.ts [hunk 2]
```diff
@@ -175,8 +175,8 @@ class SecurityEngine extends AnalysisConsumer implements DiagnosticProducer
             let diagnostic = {
                 severity: diagSeverity,
                 range: get_range(this.context.version),
-                message: this.message,
-                source: 'Dependency Analytics Plugin [Powered by Snyk]',
+                message: this.message + '\n\nView Dependency Analytics report [ctrl/cmd + shift + D]',
+                source: '\n\n Dependency Analytics plugin [Powered by Snyk]',
```
Comment (arajkumar) [inline src/consumers.ts:179]: Same as above.

File: src/consumers.ts [hunk 3]
```diff
@@ -134,8 +134,8 @@ class EmptyResultEngine extends AnalysisConsumer implements DiagnosticProducer
             return [{
                 severity: DiagnosticSeverity.Information,
                 range: get_range(this.context.version),
-                message: `Application dependency ${this.context.name.value}-${this.context.version.value} - analysis is pending`,
-                source: 'Dependency Analytics Plugin [Powered by Snyk]'
+                message: `Application dependency ${this.context.name.value}-${this.context.version.value} - analysis is pending \n\nView Dependency Analytics report [ctrl/cmd + shift + D]`,
```
[the remark this answers is not in the document]
Comment (dvandra) [reply src/consumers.ts:137]: keep a separate string in a const variable and use it in both places or just here keep it in separate string and append it here in message?

[the remark this answers is not in the document]
Comment (arajkumar) [reply src/consumers.ts:137]: >keep a separate string in a const variable and use it in both.

+1

File: src/consumers.ts (hunk 1 above)
Comment (dvandra) [reply src/consumers.ts:138]: @arajkumar 
I have tried adding `\n\n` at the end of `message` property but it adding extra space in the begging of new line (screenshot attached.)
Screenshot from 2020-06-24 16-24-54

File: src/consumers.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (dvandra) [reply src/consumers.ts]: @arajkumar PTAL

File: src/consumers.ts (hunk 1 above)
Comment (dvandra) [reply src/consumers.ts]: @arajkumar PTAL

**Comment by mathur07** [comment]
[test]

File: src/consumers.ts [hunk 4]
```diff
@@ -175,8 +178,8 @@ class SecurityEngine extends AnalysisConsumer implements DiagnosticProducer
             let diagnostic = {
                 severity: diagSeverity,
                 range: get_range(this.context.version),
-                message: this.message,
-                source: 'Dependency Analytics Plugin [Powered by Snyk]',
+                message: this.message + `\n\n${runStackReport}`,
```
Comment (arajkumar) [inline src/consumers.ts:181]:
```suggestion
                message: `${this.message} \n\n${runStackReport}`,
```

**Comment by mathur07** [comment]
@dvandra Close it for now, we will come up with something new.

**Pull request closed without merging**